ibis/expr/types/generic.py

```diff
@@ -2684,6 +2684,24 @@ def nth(self, n: int | ir.IntegerValue) -> Column:
         """
         return ops.NthValue(self, n).to_expr()
 
+    def to_list(self, **kwargs) -> list:
+        """Convert a column expression to a list.
+
+        Parameters
+        ----------
+        kwargs
+            Same as keyword arguments to [`execute`](#ibis.expr.types.core.Expr.execute)
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> t = ibis.examples.penguins.fetch().limit(5)
+        >>> t.bill_length_mm.to_list()
+        [39.1, 39.5, 40.3, None, 36.7]
+        """
+        return self.to_pyarrow().to_pylist()
```

Comment:
The kwargs need to be passed through to `to_pyarrow` here

Reply:
Ah, true, and docstring should say same as kwargs for `to_pyarrow`, too.

Reply:
Wonder why the test is passing...

I'll take a look.

Reply:
> Wonder why the test is passing...

The test logic was wrong; fixed in fe03b1d.